src/components/mol.create-user-form/create-user-form.component.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from "react";
+
+import { useCreateUser } from "../../domain/create-user.use-case";
+import { useCreateUserStrings } from "./create-user-form.string";
+import { Button } from "../atm.button/button.component";
+
+export const CreateUserForm = (): JSX.Element => {
+  const today = new Date().toISOString().split("T")[0];
+
+  const [name, setName] = useState("");
+  const [phone, setPhone] = useState("");
+  const [birthDate, setBirthDate] = useState("");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [role, setRole] = useState("");
+
+  const { newUser, loading } = useCreateUser();
+
+  function toSend(e: { preventDefault: () => void }) {
```

Comment:
por convenção usamos o prefixo `handle` para funções que são passadas pros componentes
```suggestion
  function handleSubmit(e: { preventDefault: () => void }) {
```

Reply:
ajustando